python/semantic_kernel/connectors/ai/ollama/services/ollama_base.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+from abc import ABC
+
+from semantic_kernel.kernel_pydantic import KernelBaseModel
+
+
+class OllamaBase(KernelBaseModel, ABC):
+    """Ollama service base.
+
+    Args:
+        host (Optional[str]): URL of the Ollama server, defaults to None and
+            will use the default Ollama service address: http://127.0.0.1:11434
+    """
+
+    host: str | None = None
```

Comment:
Would it be worth it to have the default Pydantic field configured with `http://127.0.0.1:11434`? Or is that happening automatically if the host is passed in as `None`?

Reply:
The SDK applies the default address automatically if it's None.